openmetadata-service/src/main/java/org/openmetadata/service/secrets/NoopSecretsManager.java

```diff
@@ -18,13 +18,14 @@
 public class NoopSecretsManager extends SecretsManager {
   private static NoopSecretsManager instance;
 
-  private NoopSecretsManager(String clusterPrefix, SecretsManagerProvider secretsManagerProvider) {
-    super(secretsManagerProvider, clusterPrefix);
+  private NoopSecretsManager(SecretsManagerProvider secretsManagerProvider, SecretsConfig secretsConfig) {
```

Comment:
Instead of calling this as NoopSecretsManager, lets call it as DBSecretsManager and use the fernet key to encrypt/decrypt the passwords.
We also needs to have config of key/value pairs thats optional and can be passed as part of the config